pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -329,16 +329,16 @@ func (p *PriorityQueue) AddUnschedulableIfNotPresent(pInfo *framework.PodInfo, p
 	defer p.lock.Unlock()
 	pod := pInfo.Pod
 	if p.unschedulableQ.get(pod) != nil {
-		return fmt.Errorf("pod is already present in unschedulableQ")
+		return fmt.Errorf("pod: %v/%v is already present in unschedulable queue", pod.Namespace, pod.Name)
```

Comment:
nit: seems that you can use the `nsNameForPod` helper method.

Reply:
done